src/__fixtures__/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export declare const bigSchemaSDL: string;
+export declare const bigSchemaIntrospectionResult: Record<string, any>;
```

Comment:
```suggestion
export declare const bigSchemaIntrospectionResult: {};
```

Reply:
You will still need to do cast inside tests but at least you will be able to get `data` without cast so it will be one cast instead of two.